tslint.json

```diff
@@ -6,14 +6,15 @@
     "trailing-comma": [false],
     "object-literal-sort-keys": false,
     "arrow-parens": false,
-    "quotemark": ["single"],
+    "quotemark": [true, "single"],
     "ordered-imports": [false],
     "variable-name": false,
     "object-literal-key-quotes": false,
     "interface-name": false,
     "member-ordering": false,
     "no-namespace": false,
-    "no-unused-expression": [true, "allow-fast-null-checks"]
+    "no-unused-expression": [true, "allow-fast-null-checks"],
+    "array-type": false
```

Comment:
Saw that we use both (Array<> and []), so wanted to silence the warnings by explicitly allowing it.